vincechain/chain.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "vincechain",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "VinceChain",
+  "chain_id": "vince_1000-1",
+  "bech32_prefix": "vce",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md